python/python-adaguc-server/routeAdagucServer.py

```diff
@@ -13,7 +13,7 @@
 logger = logging.getLogger(__name__)
 
 
-@routeAdagucServer.route("/wms", methods=["GET"])
+@routeAdagucServer.route("/wms", methods=["GET"], strict_slashes=False)
```

Comment:
Why did you have to change this?

Reply:
To make Flask accept trailing slashes (or double slashes).
Not very relevant if we go to FastApi

Reply:
Removed the strict_slashes parameter